src/custom/components/AccountDetails/Transaction.tsx

```diff
@@ -223,10 +220,10 @@ const StatusLabelWrapper = styled.div`
   justify-content: center;
 `
 
-const StatusLabel = styled.div<{ isPending: boolean }>`
+const StatusLabel = styled.div<{ isPending: boolean; isCancelling: boolean }>`
   height: 28px;
   width: 100px;
-  border: ${({ isPending, theme }) => isPending && `1px solid ${theme.border2}`};
+  border: ${({ isPending, isCancelling, theme }) => !isCancelling && isPending && `1px solid ${theme.border2}`};
```

Comment:
i would make the whole property conditional not the value. else it's still rendering: `border: false;`

```suggestion
  ${({ isPending, isCancelling, theme }) => !isCancelling && isPending && `border:  1px solid ${theme.border2};`}
```